src/Dynatrace.OpenTelemetry.Exporter.Metrics/OneAgentMetadataEnricher.cs

```diff
@@ -0,0 +1,90 @@
+// <copyright company="Dynatrace LLC">
+// Copyright 2020 Dynatrace LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace Dynatrace.OpenTelemetry.Exporter.Metrics
+{
+    /// <summary>
+    /// Queries the OneAgent to get metadata about the current process and enriches metric labels with them.
+    /// </summary>
+    public class OneAgentMetadataEnricher
```

Comment:
```suggestion
    private class OneAgentMetadataEnricher
```

Reply:
`private` would not allow us to use it anymore, switched it to `internal`